src/vs/workbench/contrib/search/browser/searchActions.ts

```diff
@@ -300,7 +300,7 @@ export function collapseDeepestExpandedLevel(accessor: ServicesAccessor) {
 				}
 				if (searchView.isTreeLayoutViewVisible && !canCollapseFirstLevel) {
 					const immediateParent = node.parent();
-					if (immediateParent instanceof FolderMatchWorkspaceRoot || immediateParent instanceof FolderMatchNoRoot) {
+					if (!(immediateParent instanceof FolderMatchWorkspaceRoot || immediateParent instanceof FolderMatchNoRoot || immediateParent instanceof SearchResult)) {
```

Comment:
Might be easier to list the positive case at this point? (I guess, instanceof Match, FileMatch, FolderMatch?)

Reply:
True! It's a little clearer for me this way, though, since it lists the node types near the root that it can't be.